.github/workflows/ci.yml

```diff
@@ -19,7 +19,7 @@ jobs:
           path: .cache
           restore-keys: |
             mkdocs-material-
-      - run: pip install git+https://${GH_TOKEN}@github.com/squidfunk/mkdocs-material-insiders.git
+      - run: pip install "mkdocs-material[imaging] @ git+https://${GH_TOKEN}@github.com/squidfunk/mkdocs-material-insiders.git"
```

Comment:
Is there a reason we don't put this in the requirements.txt instead of here?

Reply:
Old remanent of when we used the insiders version. It can be updated to portray the others. We do need to get social cards working, which requires libcairo2. I'll get an updated PR pushed.